lua/nvim-tree/view.lua

```diff
@@ -246,7 +246,7 @@ end
 local function grow()
   local starts_at = M.is_root_folder_visible(require("nvim-tree.core").get_cwd()) and 1 or 0
   local lines = vim.api.nvim_buf_get_lines(M.get_bufnr(), starts_at, -1, false)
-  local padding = 3
+  local padding =  M.View.winopts.signcolumn and 2 or 0
```

Comment:
Do we want to add an extra character (I.e. 3 instead of 3) as it was previously? I personally find the extra character useful as it indicates that I haven't reached the maximum width yet (i.e. the filename is fully shown and not clipped)

Reply:
Ah I think you got it... I didn't realise that the extra space was useful.

Yes, let's put that back and ensure that it works for max = -1 and max = 30